lib/mayaUsd/render/pxrUsdMayaGL/instancerImager.h

```diff
@@ -177,7 +179,8 @@ class UsdMayaGL_InstancerImager : public TfWeakBase {
     /// no factory has been set, returns a UsdMayaGL_InstancerShapeAdapter base
     /// class object.  The caller must manage the lifescope of the returned
     /// object.
-    static UsdMayaGL_InstancerShapeAdapter* CreateInstancerShapeAdapter();
+    static UsdMayaGL_InstancerShapeAdapter* CreateInstancerShapeAdapter(
+            const bool isViewport2);
```

Comment:
constness on the pass-by-value argument is not necessary. Can you remove it?

Regarding to coding style, we are likely to have 100 chars per line so there is no need to break a new line.

Reply:
@kxl-adsk Have we published the coding style guildline somewhere for partners?

Reply:
Another question: can this argument be true by default?

Reply:
clang-format is published on TSC forum...pending review.

Reply:
On the argument const-ness, yeah, I can remove that.

About defaulting it to true, I had thought of doing that initially, but if we made it default to true here, it seemed like it should default to true all the way down to the shape adapter constructors. It seemed safer to me to require callers to specify a value rather than accidentally end up creating a shape adapter for the wrong viewport. If you feel strongly though, I can revisit that.

Reply:
Rebased and removed the const qualifiers in f0d222549112146c7ef372041d5f9011edb5d91d.